src/plugins/betterFolders/betterFolders.css

```
@@ -0,0 +1,16 @@
+.bf-folder-sidebar [class*="wrapper-"] > div:first-of-type {
```

for consistency and possibly avoiding clashes with other mods, can you please make classes start with .vc-?
```suggestion
.vc-bf-folder-sidebar [class*="wrapper-"] > div:first-of-type {
```